cli/user/src/main/java/io/pravega/cli/user/config/InteractiveConfig.java

```diff
@@ -57,16 +57,24 @@ public class InteractiveConfig {
     private Level logLevel;
 
     public static InteractiveConfig getDefault() {
+        //Default tls based configurations for pravega cli
+        boolean tlsEnabled = false;
+        String systemEnvURI = System.getenv("PRAVEGA_CONTROLLER_URI");
```

Comment:
It could be some case that the enrironment is set to `tcp://some-pravega-uri:9090`, is it ok to leave the scheme here or we should remove it just same as `tls://` ?

Reply:
This is added for tls based configurations. To make it more generic we have issue #6819 opened.

Reply:
Personally, I think it's fine to leave the protocol there, it's a bit strange that the UI originally required no protocol to be specified.  Leaving the protocol here means the UI doesn't have to modify the provided Controller URI and backward compatibility is provided by adding the protocol if it's missing.

Obviously, there is the case where a user could specify `tcp://` for a TLS-enabled endpoint but that's no different than forgetting to set `tls-enabled` in the current CLI.  At least with this PR I can simply specify the whole Controller URI and let the CLI deal with setting itself up (and if the CA certificate is already in the JVM truststore everything will just work).

I suspect the use of `PRAVEGA_CONTROLLER_URI` environment variable will only be used in specific deployments anyway.